content/en/docs/concepts/storage/storage-classes.md

```diff
@@ -110,6 +111,27 @@ either `Delete` or `Retain`. If no `reclaimPolicy` is specified when a
 Persistent Volumes that are created manually and managed via a storage class will have
 whatever reclaim policy they were assigned at creation.
 
+### Allow Volume Expansion
+
+Persistent Volumes can be configured to be expandable. This feature when set to `true`, 
+allows the users to resize the volume by editing the corresponding PVC object. 
+
+The following types of volumes support volume expansion, when the underlying
+Storage Class has the field `allowVolumeExpansion` set to true.
+
+* gcePersistentDisk
+* awsElasticBlockStore
+* Cinder
+* glusterfs
+* rbd
+* Azure File
+* Azure Disk
+* Portworx
+* FlexVolumes
+* CSI 
```

Comment:
CSI is currently alpha

Reply:
Is 1.15 the correct feature state? Can I verify any document to make it is? I added feature state 1.15 to both the suggestions.  Please let me know if that is not correct.